doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,282 @@
+.. _tutorial_qemu:
+
+Qemu Tutorial
+*************
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   qemu-debugging.rst
+
+
+Tutorial overview
+=================
+
+In this tutorial, we will demonstrate launching an Ubuntu cloud image in a
+virtual machine that uses cloud-init to pre-configure the system during boot.
+
+The goal of this tutorial is to provide a minimal demonstration of cloud-init
+that users can use as a development environment to test cloud-init
+configurations locally prior to launching in the cloud.
+
+
+Why Qemu?
+=========
+
+Qemu is a cross-platform emulator capable of running performant virtual
```

Comment:
This is really good - I would just put a link to the Qemu website here at this first proper mention of it (for convenience)

Reply:
+1